### PR TITLE
CVP-3048: Bugfix: Fix the unbound variable error with LIST_OF_KUBEOBJECTS

### PR DESCRIPTION
**Pull request by samvarankashyap**
$subject

### CODE REVIEW

File: ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh [hunk 1]
```diff
@@ -80,25 +80,25 @@ CUSTOM_KUBEOBJECTS_PATH="${CUSTOM_KUBEOBJECTS_PATH}${KEYWORD_CUSTOM_KUBEOBJECTS}
...
-
 if [ ! -d "${CUSTOM_KUBEOBJECTS_PATH}" ]
 then
     echo "Directory ${CUSTOM_KUBEOBJECTS_PATH} DOES NOT exists."
     echo "Please check the vault"
     echo "The following step gracefully exits now"
     exit 0 
+fi
+
+echo "The ${CUSTOM_KUBEOBJECTS_PATH} found"
```
Comment (nmars) [inline ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh:91]: Suggest: `echo "CUSTOM_KUBEOBJECTS_PATH found: $CUSTOM_KUBEOBJECTS_PATH"`

File: ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh [hunk 2]
```diff
@@ -80,25 +80,25 @@ CUSTOM_KUBEOBJECTS_PATH="${CUSTOM_KUBEOBJECTS_PATH}${KEYWORD_CUSTOM_KUBEOBJECTS}
...
+echo "We find the kube_objects files inside path"
```
Comment (nmars) [inline ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh:92]: For line 92, do you mean "Looking for kube_objects inside path" ?

Comment (samvarankashyap) [reply ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh:92]: Yes,,

File: ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh (hunk 1 above)
Comment (samvarankashyap) [reply ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh:91]: Will change it.

**Pull request merged**